static/styles.css

```diff
@@ -0,0 +1,660 @@
+/* All items */
```

Comment:
To help keep our projects organized, we can create a styles directory in addition to pages directory. In the styles directory, we can have individual css files (one to correspond for each page). So you'd have a portfolio.css for example.

Doing so, keeps our css files shorter and easier to read (and eventually debug).